### PR TITLE
fix: pagination medium screens

### LINKED ISSUE #1131: Bug: pagination on highlight page look squatched
### Describe the bug

i guess theses texts shouldn't be wrapped
image

### Steps to reproduce

1. go to [highlight](https://insights.opensauced.pizza/feed)
2. scroll down
3. see error

### Affected services

insights.opensauced.pizza

### Platforms

Desktop

### Browsers

Chrome

### Environment

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by a0m0rajab**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
Refactor PaginationResults component to display entity count for better readability and user experience. The component UI has been modified to adjust for smaller screens by replacing "block" with "inline-block". The "md:block" class has been changed to "md:inline-block" to maintain visual consistency. The entity count is now displayed with a space before it for improved readability.

_description created by [OpenSauced](https://opensauced-ai.netlify.app/)_

Fixes #1131 

## Mobile & Desktop Screenshots/Recordings

Before:
image

After:
image

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [x] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by brandonroberts** [comment]
@a0m0rajab will you share some before/after screenshots?

**Comment by bdougie** [comment]
@a0m0rajab we also need an actual description, more than the issue link. I generated one with AI to test https://github.com/open-sauced/browser-extensions/pull/79

**Comment by a0m0rajab** [comment]
Added the before and after images, thank you for your feedback!
I did not write the description thinking the issue is enough! Will be more careful about it, thank you for that!

**Comment by bdougie** [comment]
> Added the before and after images, thank you for your feedback!
> 
> I did not write the description thinking the issue is enough! Will be more careful about it, thank you for that!  

The issue is never enough. We need to know your thought process and why this solution is the right solution.

**Comment by a0m0rajab** [comment]
Thank you for your feedback! I will keep it in mind next time, is there anything else I need to update for this PR?

**Comment by a0m0rajab** [comment]
Is there anything I need to do here?

**Comment by OgDev-01** [comment]
Thanks for looking into this @a0m0rajab... Can you reduce the `Go to page` button padding on the `x` axis to help keep the text in one line?.
image

Another sugestion here is to  hide the

- `Total` 
- `highlight`
on tablet screens... @bdougie and @getaheaddev thought?

File: components/molecules/PaginationResults/pagination-result.tsx [hunk 1]
```diff
@@ -19,10 +19,10 @@ const PaginationResults = ({ metaInfo, entity, total, className }: PaginationRes
       ? metaInfo.page * metaInfo.limit
       : metaInfo.itemCount;
   return (
-    <div className={`${className && className} flex items-center gap-x-1   text-sm text-light-slate-9 tracking-tight`}>
-      <span className="hidden md:block">Showing</span> <span className="text-light-slate-12">{from} -</span>
+    <div className={`${className && className} md:inline-block items-center gap-x-1   text-sm text-light-slate-9 tracking-tight`}>
+      <span className="hidden md:inline-block">Showing</span> <span className="text-light-slate-12">{from} -</span>
       <span className="text-light-slate-12">{to}</span> of {total > 999 ? humanizeNumber(total, null) : total}
-      <span>{entity}</span>
+      <span> {entity}</span>
```
Comment (OgDev-01) [inline components/molecules/PaginationResults/pagination-result.tsx:25]: We should hide this entity on medium screens only

**Review by OgDev-01 — CHANGES_REQUESTED** [review]

**Comment by bdougie** [comment]
> Sure...👍
Sorry, I was on my phone and thought this was your PR @OgDev-01. Hopefully, that explains my terse response 😅. I agree with the change

**Comment by a0m0rajab** [comment]
Fixed and did some minor updates as well, here is the result: 

https://github.com/open-sauced/insights/assets/18273833/38e74783-a915-42c6-a26b-f66d8df462b3

I did a rebase to solve conflicts.

File: components/molecules/PaginationResults/pagination-result.tsx (hunk 1 above)
Comment (a0m0rajab) [reply components/molecules/PaginationResults/pagination-result.tsx]: Thank you for the feedback, I did that in the last Commit and added a video.

**Review by OgDev-01 — APPROVED** [review]
LGTM 🍕

**Pull request merged**